src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
```

Comment:
Use a constant for the value `7` to describe the minimum number of days required to receive a long term rental discount, and another constant for the discount value `50`. This will make the code more readable and maintainable.